sdks/typescript/featurehub-javascript-client-sdk/README.md

```diff
@@ -18,6 +18,8 @@ We have deprecated [FeatureHub Eventsource Javascript SDK](https://www.npmjs.com
 ## Changelog
 
 ### featurehub-javascript-client-sdk
+#### 1.0.3
+- Minor bugfix for urls provided to connect to backend
```

Comment:
can you say: Bugfix: Edge server urls passed to the config that include '/feature' should be processed correctly

Reply:
done!